app/lib/methods/helpers/openLink.ts

```diff
@@ -38,6 +39,16 @@ const appSchemeURL = (url: string, browser: string): string => {
 };
 
 const openLink = async (url: string, theme: TSupportedThemes = 'light'): Promise<void> => {
+	const telRegExp = new RegExp(/^(tel:)/);
+	if (telRegExp.test(url)) {
+		try {
+			await Linking.openURL(url);
+			return;
+		} catch (e) {
+			log(e);
+		}
```

Comment:
I almost forgot to register here that it would be better it uses Linking.canOpenURL before the openURL, but I tested and the canOpenURL didn't work as expected for number formatted link with the android device, then I wrapped in a try/catch